lib/clients/rtm/client.js

```diff
@@ -403,13 +415,15 @@ RTMClient.prototype._handleWsMessageViaEventHandler = function _handleWsMessageV
   messageType, message) {
   var replyTo;
 
+  if (!isUndefined(this.dataStore)) {
+    this.dataStore.handleRtmMessage(this.activeUserId, this.activeTeamId, messageType, message);
+  }
+
```

Comment:
@Everesty is right, this is technically Working As Intended. A workaround could be to check [here](https://github.com/slackhq/node-slack-client/blob/update_rtm_send/lib/clients/rtm/client.js#L380) whether `message` is undefined and if `ok` is true. If so, use the timestamp provided to grab the message via `getMessageByTs` (how you would gain access to that particular method I don't know) and use that instead.